easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-foss-2017a.eb

```diff
@@ -14,7 +14,7 @@ checksums = ['43a2f9466702fb1db31df98ae6677f15']
 
 dependencies = [
     ('zlib', '1.2.11'),
-    ('Szip', '2.1'),
+    ('Szip', '2.1.1'),
```

Comment:
@jhein32 We usually don't dependencies in existing easyconfigs, but since this easyconfig was merged recently in to `develop` (cfr. #4990) and is not included in `master` yet, I consider it OK to make an exception here (especially since not bumping `Szip` here was my fault)...